dist/src/main/config/broker.standalone.yaml.template

```diff
@@ -82,6 +82,8 @@
           # This setting can also be overridden using the environment variable ZEEBE_BROKER_GATEWAY_SECURITY_AUTHENTICATION_MODE.
           # mode: none
 
+          # WARNING: These settings are deprecated! Please use the corresponding Camunda Identity
+          # properties or the environment variables defined here: https://docs.camunda.io/docs/self-managed/identity/deployment/configuration-variables/
```

Comment:
💭 Now that we're deprecating these we must also adjust the documentation https://docs.camunda.io/docs/next/self-managed/zeebe-deployment/configuration/gateway-config/#zeebegatewaymultitenancy

Reply:
I already created a PR for adjusting the docs [here](https://github.com/camunda/camunda-docs/pull/2981), but good point, I need to expand it to the MT section as well.